course5/homework05/vector2d.cc

```diff
@@ -0,0 +1,88 @@
+#include "vector2d.h"
+#include <cmath>
+
+using namespace vector;
+
+Vector2D::Vector2D(double initX, double initY) : x(initX), y(initY) {}
+
+Vector2D::Vector2D(double arg1, double arg2, System system)
+{
+    double initX;
+    double initY;
+    switch(system)
+    {
+    case System::POLAR:
+        initX = arg1 * std::cos(arg2);
+        initY = arg1 * std::sin(arg2);
+        break;
+    case System::CARTESIAN:
+        initX = arg1;
```

Comment:
I wanted to have an explicit API, so I added the default case. But the named-constructor was a much better approach.